components/Footer/index.js

```diff
@@ -1,66 +1,65 @@
 import Link from "next/link";
 import styled from "styled-components";
 import { useRouter } from "next/router";
+import Image from "next/image";
 
 export default function Footer() {
   const { pathname } = useRouter();
 
   return (
     <footer>
       <StyledNav>
-        <StyledLink href={"/"}>
-          <svg xmlns="http://www.w3.org/2000/svg" height="48" width="48">
-            <path d="M11 39h7.5V26.5h11V39H37V19.5L24 9.75 11 19.5Zm-3 3V18L24 6l16 12v24H26.5V29.5h-5V42Zm16-17.65Z" />
-          </svg>
-        </StyledLink>
         <StyledLink href={"/sleep"}>
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            height="48"
-            width="48"
-            fill={pathname === "/sleep" ? " #DAF7A6" : "black"}
-          >
-            <path d="M2 38V8.75h3v19.7h17.65V13h16.1q3 0 5.125 2.125T46 20.25V38h-3v-6.55H5V38Zm11.5-12.45q-2.25 0-3.775-1.525T8.2 20.25q0-2.25 1.525-3.775T13.5 14.95q2.25 0 3.775 1.525T18.8 20.25q0 2.25-1.525 3.775T13.5 25.55Zm12.15 2.9H43v-8.2q0-1.75-1.25-3t-3-1.25h-13.1Zm-12.15-5.9q.95 0 1.625-.675t.675-1.625q0-.95-.675-1.625T13.5 17.95q-.95 0-1.625.675T11.2 20.25q0 .95.675 1.625t1.625.675Zm0-2.3ZM25.65 16v12.45Z" />
-          </svg>
+          <StyledImage
+            className={pathname === "/sleep" ? "active" : "notActive"}
+            src={"/assets/svg/sleep.svg"}
+            alt={"Schlafen-Icon"}
+            width={45}
+            height={45}
+          />{" "}
```

Comment:
Use props to style the components dynamically✌️

```suggestion
          <StyledImage
            variant={pathname === "/sleep" ? "active" : "notActive"}
            src={"/assets/svg/sleep.svg"}
            alt={"Schlafen-Icon"}
            width={45}
            height={45}
          />
```